src/jep/InvocationHandler.java

```diff
@@ -40,6 +40,7 @@
     private long target;
 
     private Jep jep;
+    private final boolean isDirect;
```

Comment:
isDirect refers to whether or not the InvocationHandler is directly invoking the object via its __invoke__ method, or invoking the corresponding.
When a normal InvocationHandler receives a call to Runnable.run(), it'd call pythonObject.run(), but if InvocationHandler.isDirect, it'd call pythonObject() so it can be used for callable objects.